shared/styles/style-guide.desktop.js

```diff
@@ -47,6 +47,7 @@ export const globalColorsDZ2 = {
   lightGrey3: '#e0e0e0',
 
   white: '#ffffff',
+  white90: 'rgba(255, 255, 255, 0.90)',
```

Comment:
You can't use the `opacity` css prop since that affects children. If you do this it doesn't affect children and the button doesn't became transparent